userland/libtock/crt1.c

```diff
@@ -15,9 +15,11 @@ extern int main(void);
 __attribute__ ((section(".start"), used))
 __attribute__ ((weak))
 __attribute__ ((noreturn))
+__attribute__ ((naked))
```

Comment:
No `naked` here:
> While using extended asm or a mixture of basic asm and C code may appear to work, they cannot be depended upon to work reliably and are not supported. 

Naked is only for ["Basic Asm"](https://gcc.gnu.org/onlinedocs/gcc-6.3.0/gcc/Basic-Asm.html#Basic-Asm) (essentially pure assembly functions with no clobbers)